README

```diff
@@ -1,3 +1,21 @@
-Smiley icons
-============
-GNOME icon theme is distributed under the terms of either GNU LGPL v.3 or Creative Commons BY-SA 3.0 license.
+What is this?
+=============
+
+Chat is an activity used to exchange messages with friends or classmates.
+You can chat about a topic you are studying or you can share something private that happened in your life.
+
+How to use?
+===========
+
+Chat is part of the Sugar desktop.  Please refer to;
+
+* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
```

Comment:
https://sugarlabs.org points to https://wiki.sugarlabs.org/go/Get_sugar to get Sugar. Should the link or description be improved ?

Reply:
https://sugarlabs.org/ is much more generalized and comprehensive than https://wiki.sugarlabs.org/go/Get_sugar

therefore the better alternative?

Reply:
Sorry, but I think I misunderstood, comprehensive in what manner, regarding how to `Get Sugar` ?

Reply:
regarding the different ``possible options`` to get Sugar

![how-to-get-sugar](https://user-images.githubusercontent.com/16368264/40878748-f3edd1c6-66b2-11e8-8f99-0013fd9d285e.png)

Reply:
Thanks, I agree.
The link can be `Get Sugar` on the website should be removed since it is not comprehensive and similar details are shared under `SOAS` link in the first cell